packages/foreman/rubygem-hammer_cli/rubygem-hammer_cli.spec

```diff
@@ -57,12 +60,31 @@ Obsoletes: rubygem-%{gem_name}-doc < 0.3.0-2
 Documentation for %{pkg_name}
 
 %prep
-%setup -n %{pkg_name}-%{version} -q -c -T
+%{?scl:scl enable %{scl} - << \EOF}
+gem unpack %{SOURCE0}
+%{?scl:EOF}
+
+%setup -q -D -T -n  %{gem_name}-%{version}%{?prerelease}
+
 %{?scl:scl enable %{scl} - <<EOF}
```

Comment:
This shouldn't be here according to the `gem2rpm/scl.spec.erb` template. Is it really needed? If so, do we want to modify our template to allow prereleases for all gems?